src/test/java/com/commercetools/sync/commons/utils/ClientConfigurationUtilsTest.java

```diff
@@ -1,212 +1,70 @@
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
 
 class ClientConfigurationUtilsTest {
+    private static final long TIMEOUT = 20000;
+    private static final TimeUnit TIMEOUT_TIME_UNIT = TimeUnit.MILLISECONDS;
 
     @Test
-    void createClient_WithConfig_ReturnsBlockingSphereClient() {
+    void createClient_WithConfig_ReturnsSphereClient() {
         final SphereClientConfig clientConfig =
-            SphereClientConfig.of("project-key", "client-id", "client-secret");
+                SphereClientConfig.of("project-key", "client-id", "client-secret");
         final SphereClient sphereClient = ClientConfigurationUtils.createClient(clientConfig);
 
-        assertThat(sphereClient instanceof BlockingSphereClient).isTrue();
+        assertThat(sphereClient instanceof SphereClient).isTrue();
 
         assertThat(sphereClient.getConfig().getProjectKey()).isEqualTo("project-key");
     }
 
     @Test
-    void createClient_withRetryDecorator_ShouldRetryWhen502HttpResponse() {
-        final SphereClient mockSphereUnderlyingClient =
-            spy(createHttpTestDouble(intent -> HttpResponse.of(HttpStatusCode.BAD_GATEWAY_502)));
-
-        final long maxRetryAttempt = 1L;
-        final Function<RetryContext, Duration> durationFunction = retryContext -> Duration.ofSeconds(1);
-
-        final SphereClient decoratedSphereClient =
-            decorateSphereClient(mockSphereUnderlyingClient, maxRetryAttempt, durationFunction);
-
-        final CustomerUpdateCommand customerUpdateCommand = getCustomerUpdateCommand();
+    void createClient_WithConfig_ReturnsBlockingSphereClient() {
+        final SphereClientConfig clientConfig =
+            SphereClientConfig.of("project-key", "client-id", "client-secret");
+        final SphereClient sphereClient = ClientConfigurationUtils.createClient(clientConfig, TIMEOUT, TIMEOUT_TIME_UNIT);
 
-        assertThat(decoratedSphereClient.execute(customerUpdateCommand))
-            .failsWithin(2, TimeUnit.SECONDS) // first retry will be in 1 second, see: durationFunction.
-            .withThrowableOfType(ExecutionException.class)
-            .withCauseExactlyInstanceOf(BadGatewayException.class)
-            .withMessageContaining("502");
+        assertThat(sphereClient instanceof BlockingSphereClient).isTrue();
 
-        // first request + retry.
-        verify(mockSphereUnderlyingClient, times(2)).execute(customerUpdateCommand);
+        assertThat(sphereClient.getConfig().getProjectKey()).isEqualTo("project-key");
     }
 
     @Test
-    void createClient_withDefaultRetryDecorator_ShouldRetryWhen502HttpResponse() {
-        final SphereClient mockSphereUnderlyingClient =
-            spy(createHttpTestDouble(intent -> {
-                try {
-                    Thread.sleep(300);
-                } catch (InterruptedException expected) {
+    void createClient_Simulate_Stuck_SDK() {
```

Comment:
You don't need this code in here.

Reply:
I agree, it's not a test. There's no assert here.